src/sentry/incidents/models/alert_rule.py

```diff
@@ -251,24 +274,31 @@ class AlertRule(Model):
     excluded_projects = models.ManyToManyField(
         "sentry.Project", related_name="alert_rule_exclusions", through=AlertRuleExcludedProjects
     )  # NOTE: This feature is not currently utilized.
-    name = models.TextField()
-    status = models.SmallIntegerField(default=AlertRuleStatus.PENDING.value)
     # Determines whether we include all current and future projects from this
     # organization in this rule.
     include_all_projects = models.BooleanField(
         default=False
     )  # NOTE: This feature is not currently utilized.
+    name = models.TextField()
+    status = models.SmallIntegerField(default=AlertRuleStatus.PENDING.value)
     threshold_type = models.SmallIntegerField(null=True)
     resolve_threshold = models.FloatField(null=True)
     # How many times an alert value must exceed the threshold to fire/resolve the alert
-    threshold_period = models.IntegerField()
+    threshold_period = models.IntegerField(null=True)
     # This represents a time delta, in seconds. If not null, this is used to determine which time
     # window to query to compare the result from the current time_window to.
     comparison_delta = models.IntegerField(null=True)
     date_modified = models.DateTimeField(default=timezone.now)
     date_added = models.DateTimeField(default=timezone.now)
     monitor_type = models.IntegerField(default=AlertRuleMonitorTypeInt.CONTINUOUS)
     description = models.CharField(max_length=1000, null=True)
+    detection_type = models.SmallIntegerField(
+        default=AlertRuleDetectionType.STATIC, choices=AlertRuleDetectionType.choices
+    )
+    sensitivity = models.SmallIntegerField(choices=AlertRuleSensitivity.choices, null=True)
+    seasonality = models.SmallIntegerField(
+        default=AlertRuleSeasonality.AUTO, choices=AlertRuleSeasonality.choices, null=True
```

Comment:
should we default this to null?
Otherwise we could have static detection w/ auto seasonality and that doesn't make much sense? (but also hopefully shoudln't break anything🤔 )

Reply:
Yeah I could go either way on that, I can make the default null